src/components/Dashboard/TodoList.vue

```diff
@@ -81,8 +83,13 @@
                 />
               </div>
 
-              <div class="list-item__subtitle">
-                <div v-if="task.subtitle && !task.bcolErrObj" class="todo-subtitle">
+              <div v-else class="list-item__subtitle">
+                <div v-if="task.subtitle && isTypeAnnualReport(task)" class="todo-subtitle">
```

Comment:
According to the script (code), this block is only for AR, so I'm explicitly checking for this condition now.

I added a new block for IA only... on line 92 below.